Sources/Testing/Issues/Issue.swift

```diff
@@ -164,7 +197,7 @@ extension Issue: CustomStringConvertible, CustomDebugStringConvertible {
     let joinedComments = comments.lazy
       .map(\.rawValue)
       .joined(separator: "\n")
-    return "\(kind): \(joinedComments)"
+    return "\(severity): \(kind): \(joinedComments)"
```

Comment:
Suggest only including severity if it varies from the default (i.e. `!= .error`.)

Reply:
When I was considering this change, I actually concluded that it _would_ be useful to always include the severity. One reason I find it helpful is that often, people search for the string `error:` in text to find problems. Although we don't directly output this description string to the console, I think it would be generally useful for it to include "error" for issues with that severity level.

Thinking about it more now, perhaps I should go _further_ and include that word in our actual console output, for the same reason?

Reply:
That's fair. Where does this string show up? Does Xcode use it (I genuinely can't remember)? Does VS Code?

Does the string need to be title-cased? `kind` is, but now it's not going to be at the head of the string, so should it be lowercased?

Could we reorder the bits here so they fit better? Perhaps:

```swift
"\(kind) (\(severity)): \(joinedComments)"
```

Producing something like:

> An API was misused (warning): blah blah

Reply:
Moved severity later in the string, but kept it lowercase